pcdsdevices/elliptec.py

```diff
@@ -0,0 +1,62 @@
+"""
+Classes for ThorLabs Elliptec motors.
+"""
+
+
+from ophyd import Component as Cpt
+from ophyd import Device
+from ophyd.signal import EpicsSignal, EpicsSignalRO
+
+
+class ELL_Base(Device):
+    """
+    Base class for Elliptec stages.
+
+    Sufficient for control of ELL17/20 (28/60mm linear stages) and ELL14/18
+    (rotation) stages.
```

Comment:
This class looks very much like an `ophyd` `PVPositioner`. If that class fits, I suggest we use it as the base class so we can move this like a motor.

Reply:
I'll try that out and see how that works.